### PR TITLE
Rename master to control plane in the documentation

### PR DESCRIPTION
**Pull request by ViliusS**

### CODE REVIEW

**Comment by desaintmartin** [comment]
Thanks!
Regarding chart part: please bump Chart.yaml version.

**Comment by ViliusS** [comment]
Is it really needed to bump chart version? I see that it is already bumped to 6.0.8, but the latest released version is 6.0.7.

**Comment by desaintmartin** [comment]
It is needed, 6.0.8 is [already scheduled to be released](https://github.com/kubernetes/dashboard/pull/7747). ;)

**Comment by ViliusS** [comment]
Done.

File: charts/alternative.yaml [hunk 1]
```diff
@@ -211,7 +211,7 @@ spec:
       serviceAccountName: kubernetes-dashboard
       nodeSelector:
         "kubernetes.io/os": linux
-      # Comment the following tolerations if Dashboard must not be deployed on master
+      # Comment the following tolerations if Dashboard must not be deployed on control plane
```
Comment (maciaszczykm) [inline charts/alternative.yaml:214]: In this context I would change "control plane" to "control plane nodes".

**Comment by shu-mutou** [comment]
@ViliusS Thanks!
I re-ran failed jobs.

**Pull request merged**